block-library/custom-query-loop/block/style.pcss

```diff
@@ -0,0 +1,126 @@
+/* -------------------------------------------------------------------------
+ *
+ * Block - Custom Query Loop - Styles - FE / Editor
+ *
+ * ------------------------------------------------------------------------- */
+
+.wp-block-tribe-custom-query-loop {
+	--post-template-grid-template-columns: 1fr;
+	display: grid;
+	gap: var(--spacer-40);
+	grid-template-columns: var(--post-template-grid-template-columns);
+	padding: 0;
+	margin: 0;
+	list-style-type: none;
+
+	&.columns-2,
+	&.columns-4 {
+
+		@media (--mq-wp-medium) {
+			--post-template-grid-template-columns: repeat(2, minmax(0, 1fr));
+		}
+	}
+
+	&.columns-3 {
+
+		@media (--viewport-full) {
+			--post-template-grid-template-columns: repeat(3, minmax(0, 1fr));
+		}
+	}
+
+	&.columns-4 {
+
+		@media (--viewport-full) {
+			--post-template-grid-template-columns: repeat(4, minmax(0, 1fr));
+		}
+	}
+}
+
+.wp-block-tribe-custom-query-loop__pagination {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	gap: var(--spacer-20);
+	margin-top: var(--spacer-50);
+
+	.page-numbers:not(.dots, .next, .prev) {
+
+		@mixin t-body;
```

Comment:
I wonder if we should think about doing this differently. I know these are intended to be used only with Moose, but it feels a little uncomfortable to be relying on mixins and variables from a different codebase where they may change or not be present at all.